### PR TITLE
Use yarn instead of npm and fix wcjs path

### PR DESCRIPTION
**Pull request by amahdy**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/base-starter-angular/10)

### CODE REVIEW

**Comment by limonte** [comment]
Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---

*[README.md, line 41 at r1](https://reviewable.io/reviews/vaadin/base-starter-angular/10#-LDa3Tg3DuJsvSIb6d0l:-LDa3Tg40mbcX8k-F2cw:b4ju8w2) ([raw file](https://github.com/vaadin/base-starter-angular/blob/b9746edbf5068b23daa26bdf2058978e31d00689/README.md#L41)):*
> ```Markdown
> 
> ``` bash
> $ yarn install -g @angular/cli
> ```

`yarn add`, please apply the same as in https://github.com/vaadin/base-starter-vue/pull/7/files

---


*Comments from [Reviewable](https://reviewable.io/reviews/vaadin/base-starter-angular/10)*

File: README.md [hunk 1]
```diff
@@ -35,20 +35,20 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.5 with additional steps to include Vaadin components. You can recreate the project by following along these steps:
 
-First [install npm](https://docs.npmjs.com/getting-started/installing-node)
+First [install yarn](https://yarnpkg.com/lang/en/docs/install)
 
 ``` bash
-$ npm install -g @angular/cli
+$ yarn install -g @angular/cli
```
Comment (tomivirkki) [inline README.md]: This isn't a valid command

Comment (amahdy) [reply README.md]: Two commits were not merged here, fixed now.

**Comment by tomivirkki** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io/reviews/vaadin/base-starter-angular/10#-:-LDeuNeO3T-q_bPci8FT:bnfp4nl)*

**Comment by tomivirkki** [comment]
*[README.md, line 41 at r1](https://reviewable.io/reviews/vaadin/base-starter-angular/10#-LDa3Tg3DuJsvSIb6d0l:-LDezr9cEzt0fc7m8jFW:bhaq4cr) ([raw file](https://github.com/vaadin/base-starter-angular/blob/b9746edbf5068b23daa26bdf2058978e31d00689/README.md#L41)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

`yarn add`, please apply the same as in https://github.com/vaadin/base-starter-vue/pull/7/files
</blockquote></details>

I think it's fixed now

---


*Comments from [Reviewable](https://reviewable.io/reviews/vaadin/base-starter-angular/10)*

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ webcomponentsReady().then(() => {
 
 Open `src/polyfills.ts` and add webcomponents import in the end of the file:
 ``` typescript
-import '@webcomponents/webcomponentsjs/webcomponents-sd-ce.js';
+import '@webcomponents/webcomponentsjs/bundles/webcomponents-sd-ce.js';
```
Comment (limonte) [inline README.md:72]: please update `src/polyfills.ts` accordingly to this change.

File: README.md [hunk 3]
```diff
@@ -35,20 +35,20 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
...
+$ yarn add @codebakery/origami
 
-$ npm install --save @polymer/polymer@next
-$ npm install --save @vaadin/vaadin-button
-$ npm install --save @vaadin/vaadin-text-field
-$ npm i --save @webcomponents/webcomponentsjs@latest
+$ yarn add @polymer/polymer
+$ yarn add @vaadin/vaadin-button
+$ yarn add @vaadin/vaadin-text-field
+$ yarn add @webcomponents/webcomponentsjs@latest
```
Comment (limonte) [inline README.md]: no need in `@latest`

**Review by limonte — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (amahdy) [reply README.md:72]: Fixed

File: README.md (hunk 3 above)
Comment (amahdy) [reply README.md]: Fixed

**Review by limonte — APPROVED** [review]

**Comment by limonte** [comment]
Thanks for your contributions @amahdy!

**Pull request merged**